rosdep_misc.yml

```diff
@@ -0,0 +1,10 @@
+---
+# rosdep definitions for required packages that are not yet present in the official rosdep definitions
+# upstream package definitions are hosted at https://github.com/ros/rosdistro/tree/master/rosdep
+
+beignet-dev:
+  ubuntu: beignet-dev
+  debian: beignet-dev
+libcv-bridge-dev:
+  ubuntu: libcv-bridge-dev
+  debian: libcv-bridge-dev
```

Comment:
Wait, why is this necessary? There is ros-melodic-cv-brigde which is the correct package in my opininion

Reply:
Because I did not find a rosdep key for cv bridge. I look [here](https://github.com/ros/rosdistro/tree/master/rosdep) and [here](https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml). If you feel I have overlooked it, please tell me the correct rosdep key for cv bridge.

Reply:
I think you have to depend on `vision_opencv`: https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml#L13953

Reply:
Ahh I got it now. I misinterpreted how the [distribution.yaml](https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml) file works and now see that there actually is an existing `cv_bridge` key